test/e2e/infinispan/dependencies_test.go

```diff
@@ -22,6 +22,8 @@ import (
 )
 
 func TestExternalDependenciesHttp(t *testing.T) {
+	tutils.SkipForMajor(t, 13, "Donwnloading artifacts dependencies via url is broken with Infinispan 13.")
```

Comment:
@ryanemerson Do you want me to create an issue for reference?

Reply:
No I think the commit message is self explanatory.

Reply:
There's a typo here though, `s/Donwnloading/Downloading`.

Reply:
Fixed